runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/CredentialWidget/CredentialWidget.tsx

```diff
@@ -0,0 +1,55 @@
+import { useMemo, useState } from 'react';
+
+import { Edit } from '@mui/icons-material';
+import { Box, Button, Divider, Typography } from '@mui/material';
+
+import { CredentialDto } from 'runbotics-common';
+
+import { CustomCredentialSelectDialog } from './CustomCredentialSelectDialog';
+
+
+
+const CredentialWidget = () => {
+    const [open, setOpen] = useState(false);
+    const [credential, setCredential] = useState<CredentialDto | null>(null);
+    const isCustomCredential = Boolean(credential);
+    const templateType = 'mail';
```

Comment:
If it's fixed, maybe we should move it out of component and name it using uppercase letters

Reply:
It has been updated to dynamically get the value